hpat/tests/test_series.py

```diff
@@ -1693,6 +1693,14 @@ def test_impl():
         hpat_func = hpat.jit(test_impl)
         np.testing.assert_array_equal(hpat_func(), test_impl())
 
+    def test_series_unique(self):
```

Comment:
It makes sense to add addition tests, e.g. to check raising exceptions when passed arguments of incorrect type.